indispensable-cosef/src/commonMain/kotlin/at/asitplus/signum/indispensable/cosef/CoseSigned.kt

```diff
@@ -39,8 +39,10 @@ data class CoseSigned<P : Any?> internal constructor(
     val wireFormat: CoseSignedBytes,
 ) {
 
-    fun prepareCoseSignatureInput(externalAad: ByteArray = byteArrayOf()): ByteArray =
-        wireFormat.toCoseSignatureInput(externalAad)
+    fun prepareCoseSignatureInput(
+        externalAad: ByteArray = byteArrayOf(),
```

Comment:
why is this an empty bytearray and not null?

Reply:
see below